server/lib/nl/detection/date.py

```diff
@@ -35,13 +35,18 @@
     r'(in|after|on|before|since|by|until|from|between) (\d{4})(?:,)? (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)',
 ]
 
+# prep to use for LAST_YEAR dates which should include the current year.
+_LAST_YEARS_PREP = 'last_years'
 # List of date preps that indicate single date
-_SINGLE_DATE_PREPS = ['in', 'on']
+_SINGLE_DATE_PREPS = ['in', 'on', 'year']
```

Comment:
nit: `year` isn't a preposition, but do you mean for queries like: "population in year 2020" ?